docs/commands/bf.debug.md

```diff
@@ -0,0 +1,31 @@
+BF.DEBUG return more information about the bloom filter `key`.
+
+### Parameters
+
+* **key**: The name of the filter
+
+@return
+
+* **size**: Total number of items in all filters.
+* **bytes**: The bytes size of every filter.
+* **bits**: The bits size of every filter.
+* **hashes**: The number of hash functions.
+* **hashwidth**: 64-bit hash or 32-bit hash.
+* **capacity**: The entries size of every filter.
+* **size**: The number of items in the filter.
+* **ratio**: The error ratio of filter.
+
+@examples
+
+```
```

Comment:
Thank you for the contribution.
Please remove extra new lines

Reply:
Thanks~ done~